.ci_support/linux_64_c_compiler_version11c_stdlib_version2.17cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11.yaml

```diff
@@ -2,6 +2,16 @@ aws_crt_cpp:
 - 0.26.9
 aws_sdk_cpp:
 - 1.11.329
+azure_core_cpp:
+- 1.11.1
+azure_identity_cpp:
+- 1.6.0
+azure_storage_blobs_cpp:
+- 12.10.0
+azure_storage_common_cpp:
+- 12.5.0
+azure_storage_files_datalake_cpp:
+- 12.9.0
```

Comment:
You cannot set this manually, it needs to be done correctly: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6003